docs/man/man1/openvt.1

```diff
@@ -89,5 +89,4 @@ The \fI\-w\fR idea is from "sam".
 
 .SH "SEE ALSO"
 .BR chvt (1),
-.BR doshell (8),
```

Comment:
It looks like out of all the remaining commits this is the only one that makes sense.